crates/burn-jit/src/kernel/conv/conv2d/implicit_gemm.rs

```diff
@@ -0,0 +1,343 @@
+use burn_tensor::{
+    ops::{conv::calculate_conv_output_size, ConvOptions, FloatTensorOps},
+    Shape,
+};
+use cmma::{Matrix, MatrixIdent, MatrixLayout};
+use cubecl::{
+    comptime, cube,
+    ir::{Elem, FloatKind},
+    prelude::*,
+    Compiler, CubeCount, CubeDim, Feature,
+};
+use half::f16;
+
+use crate::{
+    ops::{numeric::empty_device, permute, reshape},
+    tensor::JitTensor,
+    FloatElement, IntElement, JitBackend, JitRuntime,
+};
+
+const WARP_SIZE: u32 = 32;
```

Comment:
I think it's fine to hardcode these values for now, but they should be passed to the cube kernel via a comptime struct, because the day they will change (for amd it's 64 for instance) we won't have to change the kernels, just the launch. Same for other consts below.

Reply:
Yeah I wanted to read these constants from `cubecl` ideally. I guess I'll just put them in a struct for now then so it's easier to change later.